### DISCUSSION TITLE
如何清除app的网络缓存？

### DISCUSSION THREAD

**Discussion by sibyl2116** [tw93/Pake#866] [Q&A]
release下的twitter安装后使用了测试代理，只认测试代理，换其他的全部报HSTS错误
已经清除AppData/Roming/Twitter下所有文件
无论是重装Twitter还是重启，都没法解决这个问题

**Comment by rongrong1000** [comment]
😨我也好想知道，服务器很辣鸡，浏览器偶尔都要刷新几次缓存呢，但是在pake 卡住我了

**Reply by tw93** [reply]
我去研究下，的确这个功能还挺需要在开发调试上
